### PR TITLE
FM2-249: Improve content and layout of narratives

### PR DESCRIPTION
**Pull request by ShivanshRakesh**
## Description of what I changed
Improvements that have been done:
- [x] Add details to the  properties of `Coding` type.
- [x] Add `tel:` link to properties of `ContactPoint` type when the `system` is `phone` and URL links when `system` is `url`.
- [x] Make the sub property tables reside in rows of the main property table itself.

## Issue I worked on

see https://issues.openmrs.org/browse/FM2-249

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by ShivanshRakesh** [comment]
cc @ibacher @HerbertYiga @suthagar23

**Comment by ShivanshRakesh** [comment]
I've added details for the properties of `Coding` type. Though for the properties of type `CodableConcept`, the narrative provided by HAPI is being used.

**Comment by ShivanshRakesh** [comment]
@ibacher @HerbertYiga @suthagar23 I've made all the suggested changes. PTAL.

File: omod/src/main/resources/org/openmrs/module/fhir2/narratives/Person.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
         <th:block th:if="resource.photo">
             <tr th:if="${resource.photo.url}">
                 <td>Photo:</td>
-                <td><img src="${resource.photo.url}" /></td>
```
Comment (HerbertYiga) [inline omod/src/main/resources/org/openmrs/module/fhir2/narratives/Person.html:34]: @ShivanshRakesh  Thanks for spotting this change.
The pr looks good to me. @ibacher   Hi ian,how does this entire pr look to you cc @dkayiwa

Comment (ShivanshRakesh) [reply omod/src/main/resources/org/openmrs/module/fhir2/narratives/Person.html:34]: Thanks @HerbertYiga! :)

File: omod/src/main/resources/org/openmrs/module/fhir2/narratives/AllergyIntolerance.html [hunk 2]
```diff
@@ -99,14 +99,9 @@
...
-        </tbody>
-    </table>
-    <th:block th:each="reaction: ${resource.reaction}">
-        <br/>
-        <table class="hapiPropertyTable">
-            <tbody>
+        <th:block th:each="reaction: ${resource.reaction}">
             <tr>
-                <td>Reaction:</td>
+                <td><br/>Reaction:</td>
```
Comment (ibacher) [inline omod/src/main/resources/org/openmrs/module/fhir2/narratives/AllergyIntolerance.html:104]: Is this intentional? It looks weird to me having a line break inside a `<td>`.

Comment (ShivanshRakesh) [reply omod/src/main/resources/org/openmrs/module/fhir2/narratives/AllergyIntolerance.html:104]: Actually it was looking a bit cluttered without any separation, so I added a newline. Should I remove it then?

**Comment by ShivanshRakesh** [comment]
@ibacher I've made the changes. PTAL.

**Comment by ibacher** [comment]
@ShivanshRakesh Could you post a screenshot of a sample narrative on the ticket please?

**Comment by ShivanshRakesh** [comment]
> @ShivanshRakesh Could you post a screenshot of a sample narrative on the ticket please?

Sure.

**Comment by ShivanshRakesh** [comment]
@ibacher I've added a couple of screenshots on the ticket ([link](https://issues.openmrs.org/browse/FM2-249?focusedCommentId=266053&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-266053)).

**Comment by ibacher** [comment]
Beautiful! Thanks!

**Comment by ShivanshRakesh** [comment]
Thanks @ibacher! :)

**Pull request merged**